packages/app/cypress/e2e/runner/reporter-ct-generator.ts

```diff
@@ -0,0 +1,706 @@
+import * as specLoader from './support/spec-loader'
```

Comment:
This file is mostly duplicated from `reporter.errors.cy.ts` - intent is to execute the same error validation in a CT context that is currently being done in E2E. Major changes:

1. Removed `cy.visit` tests (not valid for CT)
2. Remove `cy.route` and `cy.server` tests (long-deprecated, probably not worth spending time on for CT)
3. Replaced the `uncaught exceptions` block content to use a set of component-based errors

I wasn't able to find a good way to avoid the duplication of these tests - there are quite a few unique cases that would seriously complicate a shared version with e2e, and kept running into issues with the "mututally exclusive e2e & ct `specPattern`" behavior that prevented using the same file for both. Good news is that the underlying tests are shared which is a win